process/accountProcessor_test.go

```diff
@@ -281,3 +283,23 @@ func TestAccountProcessor_GetShardIDForAddressShouldError(t *testing.T) {
 	assert.Equal(t, uint32(0), shardID)
 	assert.Equal(t, expectedError, err)
 }
+
+func TestAccountProcessor_GetTransactionsInvalidAddrShouldErr(t *testing.T) {
+	t.Parallel()
+
+	converter, _ := factory.NewPubkeyConverter(config.PubkeyConfig{
+		Length: 32,
+		Type:   "bech32",
+	})
+	ap, _ := process.NewAccountProcessor(
+		&mock.ProcessorStub{},
+		converter,
+		database.NewDisabledElasticSearchConnector(),
+	)
+
+	_, err := ap.GetTransactions("invalidAddress")
+	assert.True(t, errors.Is(err, process.ErrInvalidAddress))
+
+	_, err = ap.GetTransactions("")
+	assert.True(t, errors.Is(err, process.ErrInvalidAddress))
```

Comment:
big :+1: for the real key converter. If you would add one more check for a valid address and the err is nil, then it would be perfect :pray: